### ISSUE TITLE
Unable to recover wallet

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#14140]
### Description
Unable to recover wallet 

### Steps to Reproduce

  1. Clean install 0.22.713
  2. Launch from console and enable payments on staging and recover wallet using an existing recovery code
  3. Console throws error

**Actual result:**
```
$ An uncaught exception occurred in the main process Uncaught Exception:
AssertionError: state must contain an Immutable.Map of 'about' 'preferences'
    at validateState (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\app\common\state\aboutPreferencesState.js:13:10)
    at Object.setRecoveryInProgress (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\app\common\state\aboutPreferencesState.js:49:13)
    at ledgerReducer (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\app\browser\reducers\ledgerReducer.js:60:41)
    at reducers.reduce (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\js\stores\appStore.js:199:24)
    at Array.reduce (<anonymous>)
    at applyReducers (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\js\stores\appStore.js:197:68)
    at handleAppAction (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\js\stores\appStore.js:264:16)
    at callbacks.forEach (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\js\dispatcher\appDispatcher.js:107:7)
    at Array.forEach (<anonymous>)
    at AppDispatcher.dispatchToOwnRegisteredCallbacks (C:\Users\bellurs\AppData\Local\Brave\app-0.22.713\resources\app.asar\js\dispatcher\appDispatcher.js:106:20)
```
Wallet never recovers

**Expected result:**
Wallet should be recovered

**Reproduces how often:**
100%

### Brave Version

**about:brave info:**

Brave | 0.22.713
-- | --
V8 | 6.6.346.26
rev | fd036ae
Muon | 6.0.9
OS Release | 10.0.16299
Update Channel | Release
OS Architecture | x64
OS Platform | Microsoft Windows
Node.js | 7.9.0
Brave Sync | v1.4.2
libchromiumcontent | 66.0.3359.139

**Reproducible on current live release:**
No

### Additional Information
Issue reproduced on Linux by @btlechowski

**Comment by kjozwiak** [comment]
Win/Linux specific. I successfully recovered 5 different wallets on `macOS 10.13.3` using `0.22.713`. @LaurenWags couldn't reproduce on macOS either.

**Comment by srirambv** [comment]
Same issue on both prod and staging wallet created

**Comment by bsclifton** [comment]
Should be fixed with new release (0.22.714)

**Comment by kjozwiak** [comment]
Verified on `Mint 18.3 x64` using the following build:

* `0.22.714` fa2fbcae947e001528bd3bd5b1c453805d419af6 
* Muon: `6.0.9`
* libchromiumcontent: `66.0.3359.139`
